.freebsd.test.py

```diff
@@ -31,13 +33,16 @@
 
 def main():
     judgeenv.env['runtime'] = {}
-    judgeenv.env['extra_fs'] = {'PERL': ['/dev/dtrace/helper$'], 'RUBY2': ['/dev/dtrace/helper$']}
+    judgeenv.env['extra_fs'] = {
+        'PERL': FilesystemPolicy([ExactFile("/dev/dtrace/helper")]),
```

Comment:
This might be a problem: `extra_fs` is supposed to be passed from the yml, and will need to be of some format that YAML can represent.

Reply:
What if we just use three keys, or change `extra_fs` to be a dictionary with three keys.